Sources/BraveWallet/Crypto/Stores/NFTDetailStore.swift

```diff
@@ -54,15 +56,18 @@ class NFTDetailStore: ObservableObject {
   @Published var isLoading: Bool = false
   @Published var nftMetadata: NFTMetadata?
   @Published var networkInfo: BraveWallet.NetworkInfo = .init()
+  var ipfsApi: IpfsAPI?
```

Comment:
nit: Can we make this a `private let` instead of `var` and move it up below `rpcService` definition